modules/nw-network-observability-operator.adoc

```diff
@@ -3,13 +3,13 @@
 
 :_content-type: PROCEDURE
 [id="nw-network-observability-operator_{context}"]
-= Network Observability Operator
+= Viewing statuses
 
-When you deploy flow collector, the Operator provides dashboards, metrics, and keeps flows accessible in the Grafana Loki log store. When a flow collector is deployed, new dashboards are available in the {product-title} console.
+The Network Observability Operator implements the Flow Collector API resource. The Flow Collector is created at the time of installation. When you deploy the Flow Collector, new dashboards and metrics are available in the {product-title} console, and flows are accessible in the Grafana Loki log store. 
```

Comment:
The verbs don't sound correct.  There is no separate step to "deploy".  Suggest:  
  
The Network Observability Operator provides the Flow Collector API. When you create a flow collector resource or instance, it deploys the pods and services to create and store network flows in the Grafana Loki log store, and displays new dashboards and metrics in the {product-title} console.